src/main/resources/db/changelog/changelog-1.yaml

```diff
@@ -0,0 +1,729 @@
+databaseChangeLog:  
+  -  changeSet:
+      id: 1
```

Comment:
Could you rename id of liqui script following such template:
20210330.[TiketNumber]-create-interviewer-table

Reply:
fixed